viewer/src/datamodels/cad/rendering/primitives.test.ts

```diff
@@ -70,7 +70,7 @@ describe('createPrimitives', () => {
   });
 
   test('no primitives doesnt return any nods', () => {
-    const nodes = Array.from(createPrimitives(emptySector, materials, bounds));
+    const nodes = Array.from(createPrimitives(emptySector, materials, bounds, null));
```

Comment:
Sending in null to a function feels bad. Any way we can have a default here?

Reply:
Fixed a default argument - I don't want to e.g. use an infinitly big bbox to represent "no filter" because that would cause unnecessary clipping operations that wouldn't cause any data reduction. And undefined is just as bad imo.